packages/ui/src/components/UserFeedbackForm/UserFeedbackForm.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  HStack,
+  IconButton,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Stack,
+  Text,
+  Textarea,
+} from '@chakra-ui/react'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { FieldErrorsImpl, useForm } from 'react-hook-form'
+import { IoSend } from 'react-icons/io5'
+import { ObjectSchema } from 'yup'
+
+import { useUserFeedbackMutation } from '@wsvvrijheid/services'
+import { UserFeedbackCreateInput } from '@wsvvrijheid/types'
+
+import { createUserFeedbackSchema } from './schema'
+import {
+  CreateUserFeedbackFormFieldValues,
+  CreateUserFeedbackFormProps,
+} from './types'
+import { FormItem } from '../FormItem'
+
+export const UserFeedbackForm: React.FC<CreateUserFeedbackFormProps> = ({
+  isOpen,
+  onClose,
+}) => {
+  const { mutateAsync } = useUserFeedbackMutation()
+  const site = window.origin
+
+  const handleUserFeedback = async (
+    data: CreateUserFeedbackFormFieldValues,
+  ) => {
+    const userFeedback: UserFeedbackCreateInput = {
+      comment: data?.comment,
+      point: data?.point as number,
+      site,
```

Comment:
It's probably best to  move window inside the function. If this component  would be rendered on the server side, it can break the app

```suggestion
      site: window.location.href,
```